tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/IcuData.java

```diff
@@ -98,48 +93,11 @@ public void replace(RbPath rbPath, RbValue rbValue) {
         add(rbPath, rbValue);
     }
 
-    public void setVersion(String versionString) {
```

Comment:
While trying to test IcuData it became clear that these methods were in the wrong place, so I added the abstract mapper base class for them.